app/src/main/java/org/simlar/helper/FileHelper.java

```diff
@@ -90,6 +91,8 @@ private static void copyFileFromPackage(final Context context, final int resourc
 			outputStream.close();
 			inputStream.close();
 			Lg.i("created ", target);
+		} catch (final FileNotFoundException e) {
```

Comment:
Maybe this?
`} catch (final FileNotFoundException | IOException e) {`
` Lg.ex(e, "Failed to create: ", target);`